internal/cli/publish.go

```diff
@@ -126,7 +126,7 @@ func PublishCmd(ctx context.Context, outputRefs string, archs []types.Architectu
 	// The build context options is sometimes copied in the next functions. Ensure
 	// we have the directory defined and created by invoking the function early.
 	bc.Options.TempDir()
-	defer os.RemoveAll(bc.Options.TempDir())
+	// defer os.RemoveAll(bc.Options.TempDir())
```

Comment:
Should this have been uncommented? 🤔

Reply:
yes, it's the cleanup and I was testing. I'll fix it.